src/main/java/nl/smallproject/www/techiteasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,48 @@
+package nl.smallproject.www.techiteasy.controllers;
+
+import nl.smallproject.www.techiteasy.models.Television;
+import nl.smallproject.www.techiteasy.services.TelevisionService;
+import org.apache.coyote.Response;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/televisions")
+public class TelevisionController {
+
+    private final TelevisionService televisionService;
+
+    public TelevisionController(TelevisionService televisionService) {
+        this.televisionService = televisionService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<Television>> getAllTelevisionsByBrand(@RequestParam(required = false) String brand) {
+        List<Television> televisions = televisionService.getAllTelevisionsByBrand(brand);
```

Comment:
"brand" is een optionele RequestParam. Wat als ik geen brand meegeef? Dan is het de bedoeling dat je de gewone "getAllTelevisions" aanroept, maar dat gebeurt hier nog niet.